### PR TITLE
Add option to pass stripHash pattern

### PR DESCRIPTION
**Pull request by bartlomiejzuber**
Works well in build produced by latest CRA.

```
with:
          repo-token: "${{ secrets.GITHUB_TOKEN }}"
          exclude: "{**/*.map,**/node_modules/**,**/*runtime*.*,**/*manifest*.*,**/*service-worker*.*}"
          pattern: "**/build/**/*.js"
          strip-hash: "\\..*chunk"
```

(delta is shows up as `20` for testing purposes (it's a map on `existing file sizes + 20`))
image

image

related issue: https://github.com/preactjs/compressed-size-action/issues/13

### CODE REVIEW

**Comment by bartlomiejzuber** [comment]
Hey @developit thank you, I'd like to merge it but I don't have access to do it. Can you do it ?

**Comment by ckybonist** [comment]
Any progress? I just need this feature.

File: action.js [hunk 1]
```diff
@@ -31,7 +42,8 @@ async function run(octokit, context) {
 	const plugin = new SizePlugin({
 		compression: getInput('compression'),
 		pattern: getInput('pattern') || '**/dist/**/*.js',
-		exclude: getInput('exclude') || '{**/*.map,**/node_modules/**}'
+		exclude: getInput('exclude') || '{**/*.map,**/node_modules/**}',
+		stripHash: stripHash(getInput('stripHash'))
```
Comment (Turbo87) [inline action.js:46]: to be consistent with e.g. `getInput('build-script')`, should we use a dasherized option name?

```suggestion
		stripHash: stripHash(getInput('strip-hash'))
```

**Comment by developit** [comment]
I will get this merged and released this week.

**Comment by bartlomiejzuber** [comment]
@developit Kindly reminder. 😄

**Comment by aulneau** [comment]
@developit could you take a look at merging this when you have a chance? Thanks :)

**Comment by developit** [comment]
This has been released in [v2](https://github.com/preactjs/compressed-size-action/releases/tag/v2).

I also made it so that any submatches in the provided regex are turned into asterisks. You can see a few examples of this in the V2 docs at the link above.

**Pull request merged**